include/onnxruntime/core/graph/graph_viewer.h

```diff
@@ -165,7 +165,8 @@ class GraphViewer {
            if a const initializer is part of the underlying Graph but not part of this GraphViewer,
            it will still be returned instead of nullptr
   */
-  const ONNX_NAMESPACE::TensorProto* GetConstantInitializer(const std::string& name, bool check_outer_scope) const;
+  const ONNX_NAMESPACE::TensorProto* GetConstantInitializer(const std::string& name,
+                                                            bool check_outer_scope = true) const;
```

Comment:
It's only during an edge case like graph validation that you do not want to check outer scope. As the graph viewer is intended to be used on a valid graph, defaulting this to true.